fab/operations/supervisor.py

```diff
@@ -188,6 +189,11 @@ def set_pillow_retry_queue_supervisorconf():
         _rebuild_supervisor_conf_file('make_supervisor_conf', 'supervisor_pillow_retry_queue.conf')
 
 
+def set_submissions_reprocessing_queue_supervisorconf():
+    if 'submission_reprocessing_queue' in _get_celery_queues():
+        _rebuild_supervisor_conf_file('make_supervisor_conf', 'supervisor_submission_reprocessing_queue.conf')
```

Comment:
sorry about that - fixed here: https://github.com/dimagi/commcare-hq-deploy/pull/350